app/services/ccms/attribute_value_generator.rb

```diff
@@ -92,6 +87,42 @@ def app_amendment_type(_options)
       @legal_aid_application.used_delegated_functions? ? 'SUBDP' : 'SUB'
     end
 
+    def client_owns_vehicle(_options)
+      @legal_aid_application.own_vehicle? ? 'true' : 'false'
+    end
```

Comment:
Do `true` and `false` have to be returned as strings? If they can be returned as Booleans, this whole section of code gets a lot easier. For example this just becomes:
```ruby
def client_owns_vehicle(_options)
  @legal_aid_application.own_vehicle?
end
```